CMakeLists.txt

```diff
@@ -214,7 +214,7 @@ install(FILES "scap-workbench.appdata.xml"
 if (ASCIIDOCTOR_EXECUTABLE)
     file(GLOB USER_MANUAL_SCREENSHOTS "${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual/*.png")
     add_custom_command(
-        OUTPUT doc/user_manual.html
+        OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual.html
```

Comment:
Well, yes and no.

On one side, it seemed reasonable to produce the output in the BINARY directory, but...

The `install`line is installing from `${CMAKE_CURRENT_SOURCE_DIR}` :

    install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual.html"

... so I assumed it made sense that `add_custom_command`'s `OUTPUT` should also point there (??)